src/main/java/frc/trigon/robot/commands/commandfactories/GeneralCommands.java

```diff
@@ -1,15 +1,17 @@
-package frc.trigon.robot.commands;
+package frc.trigon.robot.commands.commandfactories;
 
 import edu.wpi.first.wpilibj2.command.*;
 import frc.trigon.robot.RobotContainer;
-import frc.trigon.robot.constants.CommandConstants;
+import frc.trigon.robot.commands.CommandConstants;
 import frc.trigon.robot.subsystems.MotorSubsystem;
 
 import java.util.function.BooleanSupplier;
 
-public class Commands {
-    public static boolean IS_BRAKING = true;
-
+/**
+ * A class that contains the general commands of the robot, such as commands that alter a command or commands that affect all subsystems.
+ * These are different from CommandConstants because they use logic to determine what will run.
+ */
```

Comment:
These are different from (link to command constants) because they create new commands with some kind of logic, rather than only constructing existing commands with parameters.